app/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
+Django settings for app project.
+
+Generated by 'django-admin startproject' using Django 4.0.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-ms0-z4g4gyg&gk_e#9#m#8z^s9*2#x+$yn=&cq4m*d91ede^7!'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
The `DEBUG` setting is set to `True`. This should be set to `False` in a production environment to prevent the display of sensitive information in error pages.